### ISSUE TITLE
Upgrade issue on 5.73.0

### ISSUE THREAD

**Issue by DmytroLitvinov** [TryGhost/Ghost#18943]
### Issue Summary

Tried to upgrade Ghost self-hosted instance from 5.70.2 -> 5.73.0.
Firstly, I had NodeJS v16 so I had to upgrade to NodeJS v18 before upgrading to 5.73.0. I did it and then after running `ghost update` I got next issue:

```
error "components/tryghost-admin-x-design-system-5.73.0.tgz": Tarball is not in network and can not be located in cache (["/var/www/ghost/versions/5.73.0/components/tryghost-admin-x-design-system-5.73.0.tgz","/home/ghost-mgr/.cache/yarn/v6/.tmp/2916df55fe96f0c1c5e348c396ff891b/.yarn-tarball.tgz"])
```

### Steps to Reproduce

1. Upgraded NodeJS v16 -> NodeJS 18
2. Run `ghost update` to install from 5.70.2 -> 5.73.0

### Ghost Version

5.70.2

### Node.js Version

v18.17.1

### How did you install Ghost?

Debug Information:     OS: Ubuntu, v22.04.3 LTS     Node Version: v18.17.1     Ghost Version: 5.70.2     Ghost-CLI Version: 1.25.3     Environment: production     Command: 'ghost update'

### Database type

MySQL 8

### Relevant log / error output

```shell
ghost-mgr@ghostonubuntu2204-s-1vcpu-1gb-fra1-01:/var/www/ghost$ ghost update

Love open source? We’re hiring JavaScript Engineers to work on Ghost full-time.
https://careers.ghost.org



+ sudo systemctl is-active ghost_dmytrolitvinov-com
✔ Checking system Node.js version - found v18.17.1
✔ Ensuring user is not logged in as ghost user
✔ Checking if logged in user is directory owner
✔ Checking current folder permissions
✔ Checking folder permissions
✔ Checking file permissions
✔ Checking content folder ownership
✔ Checking memory availability
✔ Checking free space
✔ Checking for available migrations
✔ Checking for latest Ghost version

# 5.73.0

* ✨ Enabled emoji picker in editor (#18880) - Steve Larson
* 🐛 Fix edge case resulting in duplicate emails for some recipients (#18941) - Steve Larson
* 🐛 Fixed various editor issues (#18934) - Kevin Ansfield
* 🐛 Fixed UI glitch on unsubscribe page in Portal - Simon Backx
* 🐛 Fixed unsubscribe button requiring manual action + wrong confirmation - Simon Backx
* 🐛Changed "free_signups" to "signups" in Post Exporter (#18883) - Nick Moreton
* 🐛 Fixed email rendering bug in Gmail for Android (#18886) - Chris Raible
* 🐛 Fixed links being invisible in Portal settings (#18881) - Peter Zimon

---

View the changelog for full details: https://github.com/tryghost/ghost/compare/v5.72.2...v5.73.0

💻 We're looking for DevOps engineers to join Ghost: https://careers.ghost.org/devops-engineer

✔ Fetched release notes
✖ Downloading and updating Ghost to v5.73.0
A ProcessError occurred.

Message: Command failed: yarn install --no-emoji --no-progress
warning ghost@5.73.0: The engine "cli" appears to be invalid.
warning Resolution field "@elastic/elasticsearch@8.5.0" is incompatible with requested version "@elastic/elasticsearch@8.10.0"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@^2.29.1"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.1"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@^2.27.0"
warning Resolution field "moment-timezone@0.5.23" is incompatible with requested version "moment-timezone@^0.5.31"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.4"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.4"
warning Resolution field "moment-timezone@0.5.23" is incompatible with requested version "moment-timezone@0.5.34"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.1"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.3"
error "components/tryghost-admin-x-design-system-5.73.0.tgz": Tarball is not in network and can not be located in cache (["/var/www/ghost/versions/5.73.0/components/tryghost-admin-x-design-system-5.73.0.tgz","/home/ghost-mgr/.cache/yarn/v6/.tmp/2916df55fe96f0c1c5e348c396ff891b/.yarn-tarball.tgz"])

yarn install v1.22.19
[1/5] Validating package.json...
[2/5] Resolving packages...
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

Exit code: 1


Debug Information:
    OS: Ubuntu, v22.04.3 LTS
    Node Version: v18.17.1
    Ghost Version: 5.70.2
    Ghost-CLI Version: 1.25.3
    Environment: production
    Command: 'ghost update'

Additional log info available in: /home/ghost-mgr/.ghost/logs/ghost-cli-debug-2023-11-10T18_06_16_326Z.log

Try running ghost doctor to check your system for known issues.

You can always refer to https://ghost.org/docs/ghost-cli/ for troubleshooting.
```

### Code of Conduct

- [X] I agree to be friendly and polite to people in this repository

**Comment by DmytroLitvinov** [comment]
I see same issue at CI at releasing 5.73.0 - https://github.com/TryGhost/Ghost/actions/runs/6827053120/job/18568441359

**Comment by MxtressMyxx** [comment]
I am having the same error, but on a clean install using the supported Ubuntu stack in the production install guide

```
$ ghost install
✔ Checking system Node.js version - found v18.12.1
✔ Checking current folder permissions
✔ Checking memory availability
✔ Checking free space
✔ Checking for latest Ghost version
✔ Setting up install directory
✖ Downloading and installing Ghost v5.73.0
A ProcessError occurred.

Message: Command failed: yarn install --no-emoji --no-progress
warning ghost@5.73.0: The engine "cli" appears to be invalid.
warning Resolution field "@elastic/elasticsearch@8.5.0" is incompatible with requested version "@elastic/elasticsearch@8.10.0"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@^2.29.1"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.1"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@^2.27.0"
warning Resolution field "moment-timezone@0.5.23" is incompatible with requested version "moment-timezone@^0.5.31"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.4"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.4"
warning Resolution field "moment-timezone@0.5.23" is incompatible with requested version "moment-timezone@0.5.34"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.1"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.3"
error "components/tryghost-admin-x-design-system-5.73.0.tgz": Tarball is not in network and can not be located in cache (["/var/www/myxxdothaus/versions/5.73.0/components/tryghost-admin-x-design-system-5.73.0.tgz","/home/spectre/.cache/yarn/v6/.tmp/2916df55fe96f0c1c5e348c396ff891b/.yarn-tarball.tgz"])

yarn install v1.22.19
[1/5] Validating package.json...
[2/5] Resolving packages...
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

Exit code: 1

Debug Information:
    OS: Ubuntu, v22.04.3 LTS
    Node Version: v18.12.1
    Ghost-CLI Version: 1.25.3
    Environment: production
    Command: 'ghost install'

Additional log info available in: /home/spectre/.ghost/logs/ghost-cli-debug-2023-11-10T18_31_59_340Z.log

Try running ghost doctor to check your system for known issues.

You can always refer to https://ghost.org/docs/ghost-cli/ for troubleshooting.
```

**Comment by EtBellatorLucis** [comment]
Same issue on a stack from instructions
```
✔ Checking system Node.js version - found v18.18.2
✔ Checking current folder permissions
✔ Checking memory availability
✔ Checking free space
✔ Checking for latest Ghost version
✔ Setting up install directory
✖ Downloading and installing Ghost v5.73.0
A ProcessError occurred.

Message: Command failed: yarn install --no-emoji --no-progress
warning ghost@5.73.0: The engine "cli" appears to be invalid.
warning Resolution field "@elastic/elasticsearch@8.5.0" is incompatible with requested version "@elastic/elasticsearch@8.10.0"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@^2.29.1"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.1"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@^2.27.0"
warning Resolution field "moment-timezone@0.5.23" is incompatible with requested version "moment-timezone@^0.5.31"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.4"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.4"
warning Resolution field "moment-timezone@0.5.23" is incompatible with requested version "moment-timezone@0.5.34"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.1"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.3"
error "components/tryghost-admin-x-design-system-5.73.0.tgz": Tarball is not in network and can not be located in cache (["/var/www/ghost/versions/5.73.0/components/tryghost-admin-x-design-system-5.73.0.tgz","/var/www/.cache/yarn/v6/.tmp/2916df55fe96f0c1c5e348c396ff891b/.yarn-tarball.tgz"])

yarn install v1.22.19
[1/5] Validating package.json...
[2/5] Resolving packages...
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

Exit code: 1


Debug Information:
    OS: Ubuntu, v22.04.3 LTS
    Node Version: v18.18.2
    Ghost-CLI Version: 1.25.3
    Environment: production
    Command: 'ghost install'

Additional log info available in: /var/www/.ghost/logs/ghost-cli-debug-2023-11-10T19_45_55_405Z.log

Try running ghost doctor to check your system for known issues.

```

**Comment by diegomontanaduarte** [comment]
Same issue here!

**Comment by goxmedia** [comment]
... ditto.

**Comment by gableoley** [comment]
Same error ubuntu server node v18.18.2 yarn 1.22.19

**Comment by StuBlad** [comment]
Same issue today. Logs and details re my specific case on the Ghost forum:

https://forum.ghost.org/t/ghost-update-fails-command-failed-yarn-install-no-emoji-no-progress/42620/3

My error looks just like the OP's though.

**Comment by daniellockyer** [comment]
Thanks for the patience everyone 🙂 

This was due to an error with the automated publishing. We deprecated the release over the weekend to avoid further issues.

This should now be fixed in v5.73.1 and we're doing a portmortem internally to come up with action points to avoid this again in the future.
